draft-ietf-quic-transport.md

```diff
@@ -1829,10 +1829,24 @@ tokens that would be accepted by the server.  Only the server requires access to
 the integrity protection key for tokens.
 
 There is no need for a single well-defined format for the token because the
-server that generates the token also consumes it.  A token could include
-information about the claimed client address (IP and port), a timestamp, and any
-other supplementary information the server will need to validate the token in
-the future.
+server that generates the token also consumes it.  Tokens sent in Retry packets
+SHOULD include information that allows the server to verify that the source IP
+address and port in client packets remains constant.
```

Comment:
As you suggest in your comment, I think i would remove "port" here, as there are not that many applications where returning clients will have the same port.

Reply:
As this is just for Retry, we can include port.